### ISSUE TITLE
menu option missing

### ISSUE THREAD

**Issue by zgub** [FelisCatus/SwitchyOmega#906]
Hello

There was a fast way to use a proxy in current tab, just to click the Proxy Switchy icon and select a proxy for a wldcard directly from the menu (e.g. IP or domain), today as I started the browser the menu is somehow not showing at all. I reinstlaled proxyswitchyomega, restarted chrome but still, the "quick menu" to change proxy for just e.g. facebook.com or 192.168.0.5  is not there.

**Comment by TjWallas** [comment]
Hi,

I am also suffering from this issue.

There used to be a nice way to temporarily use proxy X for the current website Y. Also an option, to later add a permanent wildcard rule. I would love to have this feature back. 

Extension details
```
SwitchyOmega 2.3.20
```

Browser details
```
Chromium	53.0.2785.143 (Developer Build) Built on Ubuntu , running on Ubuntu 16.04 (64-bit)
Revision	12ae4ebf489873fc4aad9efc231b9151b330a938
OS	Linux 
Blink	537.36 (@12ae4ebf489873fc4aad9efc231b9151b330a938)
JavaScript	V8 5.3.332.47
Flash	(Disabled)
Chromium/53.0.2785.143 Chrome/53.0.2785.143 Safari/537.36
```

OS details
```
❯ cat /etc/lsb-release                                                                                                                                                                                
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.1 LTS"

❯ uname -a                                                                                                                                                                                           
4.4.0-45-generic #66-Ubuntu SMP x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by TjWallas** [comment]
cc @FelisCatus

**Comment by FelisCatus** [comment]
Bug confirmed. A fix is coming soon.

**Comment by FelisCatus** [comment]
This is fixed in v2.3.21. Sorry for that. I hope it didn't cause too much trouble.
